src/user/auth/signup-form/signup-form.js

```diff
@@ -142,6 +146,30 @@ class SignUpForm extends Component {
   };
 
   render() {
+    function SuccessMsg(props) {
+      return (
+        <Modal
+          {...props}
+          size="lg"
+          aria-labelledby="contained-modal-title-vcenter"
+          centered
+        >
+          <Modal.Header closeButton>
+            <Modal.Title id="contained-modal-title-vcenter">
+              SignUp Success
+            </Modal.Title>
+          </Modal.Header>
```

Comment:
@tushar912 We have defined a separate set of .scss classes for modals to be used throughout the application at https://github.com/codeuino/social-platform-donut-frontend/blob/development/src/css/components/_modals.scss 

Please have a look at these classes and add them to your component. You can find the usage of them at the other modals used throughout the application.